src/index.js

```diff
@@ -298,7 +371,7 @@ export default class Drawer extends Component {
           return (
             <div
               id={id}
-              style={{backgroundColor: `rgba(55, 56, 56, ${open ? 0.6 : 0})`}}
+              style={{ backgroundColor: `rgba(55, 56, 56, ${open ? 0.6 : 0})`, overflowX: 'hidden' }}
```

Comment:
is this needed? if it is needed, is it only needed when `isDirectionLeft` or `isDirectionRight` .. if that is the case, maybe we should only apply it when one of those is truthy

Reply:
Yes, it is needed only when `isDirectionRight` is true to hide the x-scrollbar during the sliding animation. We can set it to the default value for all the other directions.